performance/jsonParsers/myna/myna_json_parser.js

```diff
@@ -0,0 +1,8 @@
+(function (exports) {
+    var grammar = CreateJsonGrammar(Myna);
```

Comment:
where does createJsonGrammar come from?
* is it built into myna? 
* in that case can you provide to relevant source code in myna?

The reason is that users of the benchmark should be able to see how each of
the parsers was implemented.